thumbor/context.py

```diff
@@ -13,6 +13,7 @@
 from thumbor.filters import FiltersFactory
 from thumbor.metrics.logger_metrics import Metrics
 from thumbor.threadpool import ThreadPool
+import re
```

Comment:
This import should be placed before Thumbor imports.

https://github.com/thumbor/thumbor/actions/runs/6098051849/job/16586653768?pr=1587#step:11:8